test_app/after/src/sort_exported_funcs.erl

```diff
@@ -0,0 +1,19 @@
+-module(sort_exported_funcs).
+
+%% order of the function bodies is not relevant, what the format is the
+%% export list
+-format #{sort_exported_funcs => alphabetically}.
+
+-export([abc_first_function/1, second_function/2, second_function/3]).
```

Comment:
Please add tests for:
* Multiple export lists showing that each list is sorted individually.
* Functions with utf8 names (e.g. `'😅'/0`).
* Functions with uppercase names (e.g. `'EXIT'/1`).